tests/helpers.py

```diff
@@ -97,3 +102,78 @@ def max_pooling(_input, pool_shape, pool_stride):
                                pool_shape, k])
 
     return results
+
+class ResultsHelper:
+
+    def write_file(result_json, algo):
+        latest_dir = "./latest_results"
+        latest_result = latest_dir + "/progress.json"
+        res = {}
+        if osp.exists(latest_result):
+            res = json.loads(open(latest_result, 'r').read())
+        elif not osp.exists(latest_dir):
+            os.makedirs(latest_dir)
+        res[algo] = result_json
+        result_file = open(latest_result, "w")
+        result_file.write(json.dumps(res))
+
+
+    def create_json(b_csvs, g_csvs, trails, seeds, b_x, b_y, g_x, g_y, factor_g, factor_b):
```

Comment:
perhaps later you can add a `JSONOutput` to the logger? See `garage.logger`

Reply:
Agreed.  Will look into that now that logger changes has been merged.